subprojects/vutils/meson.build

```
@@ -1,4 +1,4 @@
-project('vutils', 'c', version: '0.0.0')
+project('vutils', 'c', version: '1.0.0')
```

Bump to version `0.1.0` instead, since the API has not been stabilized yet

```suggestion
project('vutils', 'c', version: '0.1.0')
```